doc/man3/EVP_RAND.pod

```diff
@@ -333,6 +333,13 @@ or the properties in the case of B<OSSL_RAND_PARAM_PROPERTIES>.
 
 =head1 NOTES
 
+The use of a nonzero value for the I<prediction_resistance> argument
+to EVP_RAND_instantiate(), EVP_RAND_generate() or EVP_RAND_reseed()
+should be used sparingly.  In the default setup, this will cause all
+public and private DRBGs to be reseeded on next use.  Since, by default,
+public and private DRBGs are allocated on a per thread basis, this can
+result in significant overhead for highly multi-threaded applications.
+
```

Comment:
I also like the formulation. But maybe it would be good to also add a recommendation after the warning?

```
For normal use-cases, the default "reseed_requests" and
"reseed_time_interval" thresholds ensure sufficient prediction resistance
over time, and you can reduce those values if you think they are too high.
Explicitly requesting prediction resistance is intended for more special
use-cases like generating long-term secrets. 
```

Reply:
Applied.